docs/site/Discovering-models-from-relational-databases.md

```diff
@@ -0,0 +1,41 @@
+---
+lang: en
+title: "Discovering models from relational databases"
+keywords: LoopBack
+sidebar: lb3_sidebar
```

Comment:
Please fix to `lb4_sidebar`.

This new file needs to be added to the sidebar, see https://github.com/strongloop/loopback-next/blob/master/docs/site/sidebars/lb4_sidebar.yml

Reply:
Done